pkg/features/kube_features.go

```diff
@@ -317,6 +317,16 @@ const (
 	// Extend the default scheduler to be aware of volume topology and handle PV provisioning
 	DynamicProvisioningScheduling utilfeature.Feature = "DynamicProvisioningScheduling"
 
+	// owner: @XiaoningDing
+	// alpha: v1.11
+	//
+	// Allow job veritical scaling
+	JobVerticalScaling utilfeature.Feature = "JobVerticalScaling"
```

Comment:
Can we please consider the name 'VerticalScaling' instead of JobVerticalScaling? Chances are community might want to extend this to other types of controllers as well.

Reply:
fixed